packages/catalog-model/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @backstage/catalog-model
 
+## 0.7.9
+
+### Patch Changes
+
+- Improve error messages when schema validation fails
```

Comment:
No need to directly edit the CHANGELOG itself.  Instead, you can use the changeset functionality.  More info here:
https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets

Reply:
Ah thank you, I have made that change.